Description: Upstream changes introduced in version 1.0.4-0ubuntu1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 libpam-phidgetrfid (1.0.4-0ubuntu1) lucid; urgency=low
 .
   * New upstream release
   * Added debian/watch file
   * Added missing postinst script
 .
 The person named in the Author field signed this changelog entry.
Author: Caleb Callaway <enlightened.despot@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ libpam-phidgetrfid-1.0.4/DEBIAN/postinst
@@ -0,0 +1,40 @@
+#!/bin/sh
+# postinst script for #PACKAGE#
+#
+# see: dh_installdeb(1)
+
+set -e
+
+# summary of how this script can be called:
+#        * <postinst> `configure' <most-recently-configured-version>
+#        * <old-postinst> `abort-upgrade' <new version>
+#        * <conflictor's-postinst> `abort-remove' `in-favour' <package>
+#          <new-version>
+#        * <postinst> `abort-remove'
+#        * <deconfigured's-postinst> `abort-deconfigure' `in-favour'
+#          <failed-install-package> <version> `removing'
+#          <conflicting-package> <version>
+# for details, see http://www.debian.org/doc/debian-policy/ or
+# the debian-policy package
+
+
+case "$1" in
+    configure)
+	pam-auth-update --package
+    ;;
+
+    abort-upgrade|abort-remove|abort-deconfigure)
+    ;;
+
+    *)
+        echo "postinst called with unknown argument \`$1'" >&2
+        exit 1
+    ;;
+esac
+
+# dh_installdeb will replace this with shell code automatically
+# generated by other debhelper scripts.
+
+
+
+exit 0
--- /dev/null
+++ libpam-phidgetrfid-1.0.4/DEBIAN/md5sums
@@ -0,0 +1,7 @@
+3881a3d913f561660fea73b0c60c759b  lib/security/pam_phidgetrfid.so
+66f65fbf80226fca6f082e0f84ee932d  usr/share/doc/libpam-phidgetrfid/changelog.Debian.gz
+124bc30ac25c6a210b2fb455a430e47b  usr/share/doc/libpam-phidgetrfid/changelog.gz
+c01aafcd201b8efdad5af45e09cb7cd6  usr/share/doc/libpam-phidgetrfid/README.gz
+0bf9b9eaa2d544361917bec8739dc578  usr/share/doc/libpam-phidgetrfid/NEWS.gz
+cae0cb24a0d568d8bb2b829c31a81577  usr/share/doc/libpam-phidgetrfid/copyright
+d0f5322477287abe0dd28dfd9f284674  usr/share/pam-configs/phidgetrfid
--- /dev/null
+++ libpam-phidgetrfid-1.0.4/DEBIAN/prerm
@@ -0,0 +1,42 @@
+#!/bin/sh
+# prerm script for #PACKAGE#
+#
+# see: dh_installdeb(1)
+
+set -e
+
+# summary of how this script can be called:
+#        * <prerm> `remove'
+#        * <old-prerm> `upgrade' <new-version>
+#        * <new-prerm> `failed-upgrade' <old-version>
+#        * <conflictor's-prerm> `remove' `in-favour' <package> <new-version>
+#        * <deconfigured's-prerm> `deconfigure' `in-favour'
+#          <package-being-installed> <version> `removing'
+#          <conflicting-package> <version>
+# for details, see http://www.debian.org/doc/debian-policy/ or
+# the debian-policy package
+
+
+case "$1" in
+    remove)
+        	pam-auth-update --package --remove phidgetrfid
+    ;;
+
+    	upgrade|deconfigure)
+	;;
+
+    failed-upgrade)
+    ;;
+
+    *)
+        echo "prerm called with unknown argument \`$1'" >&2
+        exit 1
+    ;;
+esac
+
+# dh_installdeb will replace this with shell code automatically
+# generated by other debhelper scripts.
+
+
+
+exit 0
--- /dev/null
+++ libpam-phidgetrfid-1.0.4/DEBIAN/control
@@ -0,0 +1,15 @@
+Package: libpam-phidgetrfid
+Version: 1.0.4-0ubuntu1
+Architecture: amd64
+Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
+Installed-Size: 88
+Depends: libc6 (>= 2.4), libpam0g (>= 0.99.7.1), libphidget (>= 2.1.7), libusb-0.1-4 (>= 2:0.1.12), libpam-runtime (>= 1.0.1-6)
+Recommends: phidgetrfid-utils
+Section: libs
+Priority: optional
+Homepage: http://sourceforge.net/projects/pam-phidgetrfid/
+Description: PAM authentication via Phidget RFID tag scanner
+ pam-phidgetrfid provides a PAM module that interfaces with a
+ Phidget RFID tag scanner, passing IDs scanned by the scanner
+ down the PAM stack as authentication tokens.
+Original-Maintainer: Caleb Callaway <enlightened.despot@gmail.com>
